installers/kubectl/postgres-operator.yml

```diff
@@ -206,6 +204,8 @@ spec:
                     value: "devtest"
                   - name: PGO_OPERATOR_NAMESPACE
                     value: "pgo"
+                  - name: RECONCILE_RBAC
+                    value: "true"
```

Comment:
@jmckulk Note the variable name change (`DYNAMIC_RBAC` => `RECONCILE_RBAC`) -- I don't think this affects your PR, but I wanted to make you aware.